src/Microsoft.ML.TimeSeries/SrCnnEntireAnomalyDetector.cs

```diff
@@ -359,15 +370,45 @@ public void Train(double[] values, ref double[][] results)
                 {
                     Array.Resize<double[]>(ref results, values.Length);
                 }
-                SpectralResidual(values, results, _threshold);
+
+                bool isTemporal = true;
+                double[] seriesToDetect = values.ToArray();
+
+                if (_period > 0)
+                {
+                    StlConfiguration config = new StlConfiguration(_period);
```

Comment:
>config [](start = 37, length = 6)

Can the same one be used in every call to `Train`?

Reply:
updated.

---
In reply to: [435219122](https://github.com/dotnet/machinelearning/pull/5202#discussion_r435219122) [](ancestors = 435219122)